coccinelle/oidisvalid.cocci

```diff
@@ -0,0 +1,13 @@
+@@
+expression O;
+@@
+
+- (O != InvalidOid)
++ OidIsValid(O)
```

Comment:
In the other Coccinelle checks, we also add a comment if the patch can be applied (see [this](https://github.com/timescale/timescaledb/blob/c06b647680dabac74cd027735ad7176f855c24a0/coccinelle/hash_create.cocci#L37) or [this](https://github.com/timescale/timescaledb/blob/c06b647680dabac74cd027735ad7176f855c24a0/coccinelle/hash_create.cocci#L37)). 

When the CI runs and one of the Coccinelle patches can be applied, the differences between the unpatched and the patched file are shown. The added comment makes it clearer for the user what the problem is.

Reply:
I agree please add a header to the file describing the check and a message when code does not conform so it's clear what the problem is

Reply:
Added! Thanks!